app/src/main/java/datastore/user_api/route/UserRoute.java

```diff
@@ -0,0 +1,97 @@
+package datastore.user_api.route;
+import datastore.user_api.entity.User;
+import datastore.user_api.database.UserRepository;
+import datastore.group_api.entity.Group;
+import datastore.group_api.database.GroupRepository;
+import java.util.List;
+import javax.inject.Inject;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.UriBuilder;
+import javax.ws.rs.core.UriInfo;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import javax.transaction.Transactional;
+
+@Path("/userID")
```

Comment:
The general convention is to have a hierachercal url path. So since users are members of groups, this would looks like `/groups/{group-id}/users/`

So I would suggest changing this to `/groups`.